php/BUILDING.md

```diff
@@ -1,163 +1,117 @@
 
-You can verify that PHP is loading the Ice extension by running the command
-shown below:
+By default, Ice for PHP is installed to `/opt/Ice-3.8a0`. To change the installation location, set the `prefix` Makefile
+variable:
 
 ```shell
-php -m
+make install prefix=$HOME/ice
 ```
 
-Look for `ice` among the installed modules. Note that your library search path (`LD_LIBRARY_PATH` on Linux or
-`DYLD_LIBRARY_PATH` on macOS) must include the directory containing the Ice shared libraries.
-
-Read the PHP Dependencies and PHP Source Files sections below for more information about installing the Ice extension.
+## Configuring the Ice for PHP Extension
 
-## PHP Dependencies
-
-PHP needs to locate the libraries for the Ice run-time libraries and its third-party dependencies.
-
-- Linux and macOS
+To use Ice for PHP, you must first copy the `ice.so` file from the `php` installation directory to your PHP extensions
+directory. You can determine the correct extension directory by running:
 
 ```shell
-libIce
-libIceDiscovery
-libIceLocatorDiscovery
-libIceSSL
-libbz2
+php-config --extension-dir
 ```
 
-In general, these libraries must reside in a directory of the user's (`LD_LIBRARY_PATH` on Linux or `DYLD_LIBRARY_PATH`
-on macOS). For Web servers, the libraries may need to reside in a system directory. For example, on Linux you can add
-the directory containing the Ice run-time libraries to `/etc/ld.so.conf` and run `ldconfig`.
+Next, enable the extension in your PHP configuration. Create an `ice.ini` file containing:
 
-You can verify that the Ice extension is installed properly by examining the output of the `php -m` command, or by
-calling the `phpInfo()` function from a script. For example, you can create a file in the Web server's document
-directory containing the following PHP script:
-
-```php
-<?php
-phpInfo();
-?>
+```ini
+extension=ice
 ```
 
-Then start a browser window and open the URL corresponding to this script. If the Ice extension is successfully
-installed, you will see an `ice` section among the configuration information.
-
-## PHP Source Files
-
-In addition to the binary Ice extension module and its library dependencies, you will also need to make the Ice for PHP
-source files available to your scripts. These files are located in the `lib` subdirectory and consist of the Ice run
-time definitions (`Ice.php`) along with PHP source files generated from the Slice files included in the Ice
-distribution.
-
-The Ice extension makes no assumptions about the location of these files, so you can install them anywhere you like.
-For example, you can simply include them in the same directory as your application scripts. Alternatively, if you
-prefer to install them in a common directory, you may need to modify PHP's `include_path` directive so that the PHP
-interpreter is able to locate these files. For example, you could append to `php.ini`:
+Place this file in one of PHP's `.ini` scan directories. You can inspect your PHP configuration with:
```

Comment:
I wouldn't describe it that way. What about:
```
Place this file in your PHP configuration file directory, often called `conf.d`.
You can inspect your PHP configuration with:
...
```

Reply:
Fixed as suggested